roles/falco/README.rst

```diff
@@ -0,0 +1,40 @@
+This ansible role installs Falco.
+
+**Role Variables**
+
+.. zuul:rolevar:: falco_configuration_files
+   :default: netdata.conf
```

Comment:
should looks like this 
/etc/falco/falco.yaml
/etc/falco/*.yaml

Reply:
This Pull Request is only for adding the documentation. Please open a new issue for changing default values.